src/main/java/robosky/uplands/client/render/entity/EntityRenderingRegistry.java

```diff
@@ -0,0 +1,27 @@
+package robosky.uplands.client.render.entity;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.client.rendereregistry.v1.EntityRendererRegistry;
+import net.minecraft.client.render.entity.MobEntityRenderer;
+import net.minecraft.entity.EntityType;
+import robosky.uplands.entity.EntityRegistry;
+
+@Environment(EnvType.CLIENT)
+public class EntityRenderingRegistry {
+    public static void init() {
+        register(EntityRegistry.HEXAHAEN, HexahaenRender.class);
+    }
+
+    private static void register(EntityType<?> entity, Class<? extends MobEntityRenderer<?, ?>> renderer) {
+        EntityRendererRegistry.INSTANCE.register(entity, (entityRenderDispatcher, context) -> {
+            MobEntityRenderer<?, ?> render = null;
+            try {
+                render = renderer.getConstructor(entityRenderDispatcher.getClass()).newInstance(entityRenderDispatcher);
```

Comment:
This is weird. Why not just pass like `(dispatcher, ctx) -> new HexahaenRender(dispatcher)`.

No need to use reflection to register entity renderers.

Reply:
Because if I use it in a lambda expression, it doesn't recognize the renderer argument for the method.

Reply:
Maybe look a little more into it? Definitely not gonna like reflection in order to do that

Reply:
If you can't figure it out yet, throw a `fix me` comment right above the entry

Reply:
Alright, sorry I suck so much lol

Reply:
I'll figure it out later.